react/Modal/Readme.md

```diff
@@ -34,6 +47,25 @@ const sizes = [
 </div>
 ```
 
+### Space
+
+Besides the default spacing inside a Modal, you can choose another type from this two avalaible: `compressed` and `spacious`
+
+```
+initialState = { modalDisplayed: false};
+const spaces = [
+  'compressed',
+  'spacious',
+];
+
+<div>
+  {spaces.map(space => <button onClick={()=>setState({ space, modalDisplayed: !state.modalDisplayed, })}>
+    { space }
+  </button>)}
+  {state.modalDisplayed && <Modal title={ state.space + ' modal'} space={state.space} description={content.ada.short} dismissAction={() => setState({ modalDisplayed: false })} /> }
```

Comment:
I would call this property `spacing` and use `large` and `small` instead of `compressed` and `spacious`. I think it is simpler and will be memorized more easily.

Reply:
I had a hard time finding the right naming 😆 but I like yours better.